packages/checkbox/src/react/__specs__/index.spec.tsx

```diff
@@ -16,6 +16,12 @@ describe('Checkbox', () => {
     expect(ref).not.toBeNull()
   })
 
+  it('composes className', () => {
+    const { container } = render(<Checkbox className="compose-classname" />)
+
+    expect(container.firstChild).toHaveClass('psds-checkbox compose-classname')
+  })
+
```

Comment:
Nice got to remember to add this test others myself

Reply:
+1 yes, it's in the checklist in #1819.